### ISSUE TITLE
STP Engine - Token infotons saved incorrectly, frequently several times in same second 

### ISSUE THREAD

**Issue by yanpom** [thomsonreuters/CM-Well#671]
Look at the example below, token infoton saved several times during 1 second, the number of consumed infotons is the same:
```
    "type" : "ObjectInfoton",
    "system" : {
      "uuid" : "90cf8090f78f36fd8df2d399b3ece948",
      "lastModified" : "2018-05-06T09:23:40.945Z",
      "path" : "/meta/sys/agents/sparql/oafullSubType/tokens/oafullSubTypeoa-relationship",
      "dataCenter" : "egl1",
      "indexTime" : 1525598621007,
      "parent" : "/meta/sys/agents/sparql/oafullSubType/tokens"
    },
    "fields" : {
      "token" : [ "GACFeJxtyz0OwjAMQOHLsBInhSTQrQsjAzewglEi5U-xO4B6eFSxIr3xe-bsppN3F2u18xs8UVBJbIVbHbQKDVahlU3endQhf-7mtsxRpM_wsxwiFWT1ShVrSJj_3FBIcNcwaX097hl4UEZJrXJM_QvykS6I" ],
      "receivedInfotons" : [ "696459" ]
    }
  }, {
    "type" : "ObjectInfoton",
    "system" : {
      "uuid" : "3f789dc0ef3ab6e23485b83f2447bc81",
      "lastModified" : "2018-05-06T09:23:40.968Z",
      "path" : "/meta/sys/agents/sparql/oafullSubType/tokens/oafullSubTypeoa-relationship",
      "dataCenter" : "egl1",
      "indexTime" : 1525598621010,
      "parent" : "/meta/sys/agents/sparql/oafullSubType/tokens"
    },
    "fields" : {
      "token" : [ "-gCFeJxtyz0OgzAMQOHLdCUmgQJl68LI0BtYwSiR8ifsDiAOXyHWSm_8nm470_Td8NSDaU5YUFCJy5Fz2ugrtLGyOZ6yF1KPcMx6eo9OpIxwW7aOIrJafcJkPYY_N0QSvDSYun5VVxo-FFB8Tux8-QHwrS6E" ],
      "receivedInfotons" : [ "696459" ]
    }
  }, {
    "type" : "ObjectInfoton",
    "system" : {
      "uuid" : "727debb6b39da0a1947fcb4d0ba79f89",
      "lastModified" : "2018-05-06T09:23:40.987Z",
      "path" : "/meta/sys/agents/sparql/oafullSubType/tokens/oafullSubTypeoa-relationship",
      "dataCenter" : "egl1",
      "indexTime" : 1525598621011,
      "parent" : "/meta/sys/agents/sparql/oafullSubType/tokens"
    },
    "fields" : {
      "token" : [ "rQCFeJxtyzkOgzAQQNHL0MaDISymo6GkyA1GzkS25E3MUIA4fITSRvrl-_rZN-3Qj11nBnPBGwWVuBw5p412oY2VzfGSo5CqwrnqZZ6cSJngZ9k6isjq4xMm6zH8uSGS4K2hqWvzuNPwooDic2Lnyxf4UC6U" ],
      "receivedInfotons" : [ "696459" ]
    }
```

**Comment by nicklester** [comment]
Fixed in PR #692

**Comment by yanpom** [comment]
Not relevant after moving to zstore 
Closing
